src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -1517,8 +1517,6 @@ internal static string GetRunAsVirtualAccountGroupsString(string[] groups)
         internal static string GetWinrmPluginShellName()
         {
             // PowerShell Core uses a versioned directory to hold the plugin
-            // TODO: This should be PSVersionInfo.PSVersionName once we get
```

Comment:
Does the removal of this TODO mean we no longer intend for this to happen?

Reply:
Removal of this comment means that the current design supports semver (pre-release labels)